test/account.spec.ts

```diff
@@ -325,40 +412,8 @@ describe('privateToAddress', function() {
   it('should produce an address given a private key', function() {
     const address = '2f015c60e0be116b1f0cd534704db9c92118fb6a'
     // Our private key
-    const privateKey = Buffer.from([
-      234,
-      84,
-      189,
-      197,
-      45,
-      22,
-      63,
-      136,
-      201,
-      58,
-      176,
-      97,
-      87,
-      130,
-      207,
-      113,
-      138,
-      46,
-      251,
-      158,
-      81,
-      167,
-      152,
-      154,
-      171,
-      27,
-      8,
-      6,
-      126,
-      156,
-      28,
-      95,
-    ])
+    // prettier-ignore
+    const privateKey = Buffer.from([234, 84, 189, 197, 45, 22, 63, 136, 201, 58, 176, 97, 87, 130, 207, 113, 138, 46, 251, 158, 81, 167, 152, 154, 171, 27, 8, 6, 126, 156, 28, 95,])
```

Comment:
Nit: convert to hex string

Reply:
done in https://github.com/ethereumjs/ethereumjs-util/pull/275/commits/6c89111e4b60b4fd3a1d98205199d6d274b61959